### PR TITLE
[ADD] hr_employee_family: Adding module to add family

### PR DESCRIPTION
**Pull request by luistorresm**
Added models to set relatives to employee

### CODE REVIEW

File: hr_family/i18n/es.po [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+#: field:hr.family,create_uid:0
+#: field:hr.family.type,create_uid:0
+msgid "Created by"
+msgstr "Creado por"
+
+#. module: hr_family
+#: field:hr.family,create_date:0
+#: field:hr.family.type,create_date:0
+msgid "Created on"
+msgstr "Creafo en"
```
Comment (jorgenaranjoVauxoo) [inline hr_family/i18n/es.po]: Creafo?

File: hr_family/security/.~lock.ir.model.access.csv# [hunk 2]
```diff
@@ -0,0 +1 @@
+luistorres ,luistorres,luistorres-ThinkPad-Edge,27.10.2014 18:31,file:///home/luistorres/.config/libreoffice/4;
```
Comment (jorgenaranjoVauxoo) [inline hr_family/security/.~lock.ir.model.access.csv#]: ??????

File: hr_family/security/hr_family_type_security.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (jorgenaranjoVauxoo) [inline hr_family/security/hr_family_type_security.xml]: Este grupo porque se agrego? Ya estiste HR Manager

**Comment by moylop260** [comment]
:+1:

**Pull request merged**